ddtrace/internal/processor/trace.py

```diff
@@ -56,9 +56,18 @@ class TraceSamplingProcessor(TraceProcessor):
     parts of the trace are unsampled when the whole trace should be sampled.
     """
 
+    _compute_stats_enabled = attr.ib(type=bool)
```

Comment:
yes! i like the idea of keeping the processors independent

Reply:
Going to hold off on this one just because it's easy to change after the fact and it would involve a bit more machinery to implement. If the different configurations diverge more then I'd say it'd be necessary to separate the logic into its own processor.